### DISCUSSION TITLE
I can't seem to start it using Pop_OS 20.04

### DISCUSSION THREAD

**Discussion by hinzwifi** [tmiland/invidious-updater#67] [Q&A] [answered]
I am just knew to this type of things and I was excited to install invidious on my Pop_OS but then it said that it doesn't know my distro so it can't download. I thought Pop_OS was a branch of ubuntu.

**Comment by tmiland** [comment, marked as answer]
Please test latest Master dbc6642

**Reply by hinzwifi** [reply]
oh okay2 thank you. I'll try it when I arrive at home 
Thank you
